DiscordRPC/DiscordRPC.nuspec

```diff
@@ -36,9 +44,21 @@ This is an unoffical, non-discord supported library.
     <file src="./bin/Release/netstandard2.0/DiscordRPC.dll" target="lib\netstandard2.0" />
     <file src="./bin/Release/netstandard2.0/DiscordRPC.pdb" target="lib\netstandard2.0" />
     <file src="./bin/Release/netstandard2.0/DiscordRPC.xml" target="lib\netstandard2.0" />
-	
-	<file src="./bin/Release/net45/DiscordRPC.dll" target="lib\net45" />
-    <file src="./bin/Release/net45/DiscordRPC.pdb" target="lib\net45" />
-    <file src="./bin/Release/net45/DiscordRPC.xml" target="lib\net45" />
+
```

Comment:
This is a good idea. This sort of addition to add more support for different runtimes is welcomed :)